### DISCUSSION TITLE
Mandatory argument <em>commissure_coordinates</em> of acpcOrNormalization_5 has no value

### DISCUSSION THREAD

**Discussion by fernand-f** [brainvisa/brainvisa.github.io#105] [Q&A]
Hello,

I'm new to BrainVisa and I need to use it in order to segment sulcus in a brain structure. I tried to run the pipeline on one of my brain data, but I got this runtime error : 

RuntimeError: Mandatory argument <em>commissure_coordinates</em> of acpcOrNormalization_5 has no value

Any idea how to fix it?

Thank you in advance.

**Comment by c-langlet** [comment]
Hi,

What pipeline are you trying to run ? I guess according to your error that at this point you are trying to run the AC/PC Or Normalization pipeline but is it part of another bigger pipeline or are you trying to run it directly ? 

Your error means that the argument _commissure_coordinates_ is not specified : 
- If you are using the graphical version, use the database button (green/red if it is input/output) to use one of the Brainvisa suggestions, or directly specify a filename (with a proper format extension, _.APC_ in this case).
- If you are trying to run it with axon-runprocess (CLI), you may need to add the flag --enabledb to make sure that the completion is set up

**Comment by denisri** [comment]
I guess you are using the Morphologist pipeline. AC/PC points are used whenever no automatic normalization process can be used. BrainVisa does not provide very accurate and robust normalization algorithms and relies on external software for this. The recommended solution is to use SPM: you may install SPM standalone and configure it in BrainVisa (in the prefences) so that it can use it. Alternatively the "AimsMIRegister" is a rigid "low-cost" alternative but does not provide the same robustness or accuracy.
Otherwise the AC/PC points (anterior and posterior commissure) may be selected manually to avoid the normalization step.
